src/app/App.tsx

```diff
@@ -136,31 +134,28 @@ export const App = () => {
 	/* istanbul ignore next */
 	const storage = isE2E() || isUnit() ? new StubStorage() : "indexeddb";
 
-	const [env] = useState(
-		() =>
-			new Environment({
-				coins: {
-					// ADA,
-					ARK,
-					ATOM,
-					AVAX,
-					BTC,
-					DOT,
-					ETH,
-					EGLD,
-					LSK,
-					NEO,
-					NANO,
-					LUNA,
-					TRX,
-					XLM,
-					XRP,
-					ZIL,
-				},
-				httpClient,
-				storage,
-			}),
-	);
```

Comment:
here we're using [lazy initial state](https://reactjs.org/docs/hooks-reference.html#lazy-initial-state)
to be executed only on the initial render, and it's better to keep it

Reply:
Didn't know about that, thanks